src/components/FormLabel/FormLabel.scss

```diff
@@ -1,21 +0,0 @@
-@import '../../styles/variables.scss';
-
-.Palmetto-FormLabel {
-  font-size: $size-font-sm;
-  color: $color-font-base;
-  display: block;
-  font-weight: 700;
-
-  &.error {
-    color: $color-brand-danger-base;
-  }
-
-  +input[type=text],
-  +input[type=tel],
-  +input[type=password],
-  +input[type=search],
-  +input[type=url],
-  +.selectInput {
-    margin-top: $size-spacing-xs;
-  }
```

Comment:
This isn't needed anymore or is it just moved somewhere else that I'm not seeing?

Reply:
@juanfabrega This spacing is defiantly absent; border-radius is missing as well.

<img width="455" alt="Screen Shot 2020-07-22 at 9 45 17 AM" src="https://user-images.githubusercontent.com/5695990/88204468-24f00f00-cc00-11ea-9504-0722def90076.png">

Reply:
@RyanMacBern @JCarmine I re-structured these so that the Inputs actually pass a className prop to the label. That way the label is not necessarily concerned with its context and instead the context governs the label.

Can y'all do a `yarn` to make sure you've got the latest tokens package locally? The inputs now pass a utility class but the previous version of tokens did not have these classes. Here is where that class is passed in the `TextInput` component:

```
const labelProps = {
    isFieldRequired: isRequired,
    inputId: id,
    labelText: label,
    hasError: !!error,
    className: 'm-bottom-xs',
  };
```

Again I think this is a better architectural pattern since we don't have non-explicit pollution into our input styles from an external component.

Reply:
Remember, Chromatic picks up visual changes, so a change like this would be caught (and would have to be approved). You can always view the PR storybook that's created by the CI. 

<img width="883" alt="Screen Shot 2020-07-22 at 10 23 30 AM" src="https://user-images.githubusercontent.com/1447339/88208005-70f18280-cc05-11ea-9dd8-4d87f41134d6.png">